stdlib/public/Cxx/CxxSet.swift

```diff
@@ -13,11 +13,24 @@
 public protocol CxxSet<Element> {
   associatedtype Element
   associatedtype Size: BinaryInteger
+  associatedtype InsertionResult // std::pair<iterator, bool>
+
+  init()
+
+  @discardableResult
+  mutating func __insertUnsafe(_ element: Element) -> InsertionResult
 
   func count(_ element: Element) -> Size
 }
 
 extension CxxSet {
+  public init<S: Sequence>(_ sequence: S) where S.Element == Element {
+    self.init()
+    for item in sequence {
+      self.__insertUnsafe(item)
+    }
+  }
+
   public func contains(_ element: Element) -> Bool {
```

Comment:
also, should this be @inlinable for perf?

Reply:
Thanks, added `@inlinable` in https://github.com/apple/swift/pull/65139.